### PR TITLE
Ingest MITPress journals through 2022-09

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

File: data/xml/2022.tacl.xml [hunk 1]
```diff
@@ -662,5 +662,118 @@
...
+      <pages>904–921</pages>
+      <url hash="89ccf909">2022.tacl-1.53</url>
+      <bibkey>agarwal-nenkova-2022-temporal</bibkey>
+    </paper>
+    <paper id="54">
+      <title>Learning <fixed-case>E</fixed-case>nglish with Peppa Pig</title>
```
Comment (nschneid) [inline data/xml/2022.tacl.xml:734]: `<fixed-case>P</fixed-case>eppa <fixed-case>P</fixed-case>ig`

File: data/xml/2022.tacl.xml [hunk 2]
```diff
@@ -662,5 +662,118 @@
...
+      <pages>956–980</pages>
+      <url hash="c3e6a886">2022.tacl-1.56</url>
+      <bibkey>naik-etal-2022-adapting</bibkey>
+    </paper>
+    <paper id="57">
+      <title>How to Dissect a Muppet: The Structure of Transformer Embedding Spaces</title>
```
Comment (nschneid) [inline data/xml/2022.tacl.xml:768]: `<fixed-case>M</fixed-case>uppet`

File: data/xml/2022.cl.xml [hunk 3]
```diff
@@ -222,4 +222,90 @@
...
+      <pages>491–516</pages>
+      <url hash="88a62714">2022.cl-3.1</url>
+      <bibkey>alemany-puig-ferrer-i-cancho-2022-linear</bibkey>
+    </paper>
+    <paper id="2">
+      <title>The Impact of Edge Displacement Vaserstein Distance on <fixed-case>UD</fixed-case> Parsing Performance</title>
```
Comment (nschneid) [inline data/xml/2022.cl.xml:244]: `<fixed-case>V</fixed-case>aserstein`

File: data/xml/2022.cl.xml [hunk 4]
```diff
@@ -222,4 +222,90 @@
...
+      <pages>635–672</pages>
+      <url hash="b19e1a1e">2022.cl-3.5</url>
+      <bibkey>choenni-shutova-2022-investigating</bibkey>
+    </paper>
+    <paper id="6">
+      <title>Survey of Low-Resource Machine Translation</title>
+      <author><first>Barry</first><last>Haddow</last></author>
+      <author><first>Rachel</first><last>Bawden</last></author>
+      <author><first>Antonio Valerio Miceli</first><last>Barone</last></author>
```
Comment (danielgildea) [inline data/xml/2022.cl.xml:290]: last name Miceli Barone

File: data/xml/2022.cl.xml [hunk 5]
```diff
@@ -222,4 +222,90 @@
...
+      <pages>517–554</pages>
+      <url hash="54be90e9">2022.cl-3.2</url>
+      <bibkey>anderson-gomez-rodriguez-2022-impact</bibkey>
+    </paper>
+    <paper id="3">
+      <title><fixed-case>UD</fixed-case>apter: Typology-based Language Adapters for Multilingual Dependency Parsing and Sequence Labeling</title>
+      <author><first>Ahmet</first><last>Üstün</last></author>
+      <author><first>Arianna</first><last>Bisazza</last></author>
+      <author><first>Gosse</first><last>Bouma</last></author>
+      <author><first>Gertjan van</first><last>Noord</last></author>
```
Comment (danielgildea) [inline data/xml/2022.cl.xml:258]: last name van Noord

**Review by danielgildea — APPROVED** [review]

**Pull request merged**